scripts/normalize-by-median.py

```diff
@@ -209,12 +208,12 @@ def normalize_by_median_and_check(input_filename, htable, single_output_file,
         total = norm.total
```

Comment:
originally they made things neat for the diagnostics that used to be right after those lines. When I moved the diags I forgot about them.